legacy/messaging/src/main/resources/schema/jboss-as-messaging_1_4.xsd

```diff
@@ -108,6 +108,13 @@
                   </xs:documentation>
               </xs:annotation>
           </xs:element>
+          <xs:element maxOccurs="1" minOccurs="0" name="override-in-vm-security" type="xs:boolean">
```

Comment:
Yes, that was a bug in the schema. But 1.4 has already been released and we can't change it.
We've added the 1.5 XSD just with just that additional element and used the 1.4 parse to parse both 1.4 and 1.5: https://issues.jboss.org/browse/WFLY-4533